MiniApp/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,11 @@
+"error_title" = "Error";
+"error_list_message" = "Couldn't retrieve Mini App list, please try again later";
+"error_single_message" = "Couldn't retrieve Mini App info, please try again later";
+"error_incorrect_appid_message" = "Incorrect Mini App ID, please try again";
+"error_miniapp_message" = "Couldn't retrieve Mini App, please try again later";
+
+"input_miniapp_title" = "Please enter Mini App ID";
+
+"wait_message" = "Please wait...";
+"OK" = "OK";
```

Comment:
I think the `key` can still be lower case 😄

Reply:
OK 😂